lesson1_7.py

```diff
@@ -0,0 +1,9 @@
+# Напишите программу, которая преобразует строку,
+# в которой записаны слова через “,” в
+# список из этих слов
+from typing import List
+
+if __name__ == '__main__':
+    some_string = 'aaa,bbb,ccc,ddd,eee'
+    new_list: List[str] = some_string.split(',')
```

Comment:
type hint це прикольно, але було б достатньо просто спліт використати
```suggestion
    new_list = some_string.split(',')
```